assets/components/src/button/style.scss

```diff
@@ -2,373 +2,10 @@
  * Button
  */
 
-@import '../../../shared/scss/colors';
-@import '~@wordpress/base-styles/colors';
-
-@mixin button-style( $background, $border, $text ) {
-	background: $background;
-	border-color: $border;
-	color: $text;
-}
-
-body[class*='admin-color-'] {
-	.newspack-button {
-		border-radius: 3px;
-		font-size: 14px;
-		font-weight: bold;
-		height: auto;
-		line-height: 24px;
```

Comment:
The reduced/default line height for buttons causes some Action Card components to look a bit squished. Maybe we need some margin for these?

<img width="891" alt="Screen Shot 2022-06-09 at 12 57 48 PM" src="https://user-images.githubusercontent.com/2230142/172927491-47e479aa-a942-42b1-9914-9b4391f74747.png">

Reply:
Fixed: https://github.com/Automattic/newspack-plugin/pull/1676/commits/789dc0f6d602a9502358f42c4662020880701412

Reply:
I like the option to make the secondary button `destructive`, too 👍